### ISSUE TITLE
Have local data instead of online

### ISSUE THREAD

**Issue by jean-michel-gonet** [jean-michel-gonet/angular-backtest#1]
# Why?
Fetching online data is nice because you always have it fresh, but there are some shortcomings:
- Online data are usually not meant to be consumed externally, so the format can change without notice, and your application will stop working.
- Most of online data requires that you login to a banking system, and those are heavily defended against data retrieval from third party applications.
- Typescript, Javascript and Internet security are very worried when they see a script from one website trying to access data from another website. 

The bottom line is, fetching online data has too many disadvantages.

# Condition of satisfaction
- Keep data in separate folders, using the name of the provider and the instrument ISIN, so they can be found easily.
- Keep the data untouched, just saved from the provider.
- Implement the mapping so the data can be used as native ``Stock`` and ``StockData``.
